Gemfile

```diff
@@ -22,7 +22,7 @@ gem "select2-rails"
 gem "sentry-sidekiq"
 gem "sprockets-rails"
 gem "terser"
-gem "validates_timeliness", "~> 7.1.0"
+gem "validates_timeliness", "~> 8.0.0"
```

Comment:
TODO (by whoever works on this Rails upgrade): look into why this gem was pinned to a specific version.

If there's no obvious reason, we should remove the version constraint from the Gemfile.